aiohttp_devtools/runserver/watch.py

```diff
@@ -53,19 +53,21 @@ async def _run(self, live_checks=20):
         try:
             self._start_dev_server()
 
+            static_path = str(self._app['static_path'])
             async for changes in self._awatch:
                 self._reloads += 1
+                is_static = all(str(c[1]).startswith(static_path) for c in changes)
```

Comment:
this doesn't need to be calculated until it's used, maybe move it inline, (perhaps via a separate function).

Reply:
Is a lambda fine? Just trying to keep the `is_static` name for readability.

Reply:
OK, linter says no.